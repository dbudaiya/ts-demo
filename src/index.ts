export function add(...args: number[]) {
  return args.reduce((a, b) => a + b, 0)
}

export const result = 'footer'

export function timeout() {
  return 'this is data'
}

declare const decLiteral: number;
