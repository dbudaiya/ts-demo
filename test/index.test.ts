import { describe, expect, it } from 'vitest'
import { add } from '../src/index'

describe('should', () => {
  it('exported', () => {
    expect(1).toEqual(1)
  })
})

describe('test function', () => {
  it.concurrent('add', () => {
    expect(add(1, 2, 3)).toEqual(6)
  })
})
