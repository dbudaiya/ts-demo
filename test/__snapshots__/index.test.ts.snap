// Vitest Snapshot v1

exports[`1 > 使用快照 > header 1`] = `"laji"`;

exports[`1 > 使用快照 1`] = `"footer"`;
